Dockerfile

```diff
@@ -13,10 +13,10 @@ RUN apk add --no-cache ca-certificates curl jq \
 FROM quay.io/giantswarm/alpine:3.11-giantswarm as base
 
 USER root
-RUN apk add --no-cache git
+RUN apk add --no-cache git jq
 
 USER giantswarm
 COPY --from=binaries /binaries/* /usr/bin/
-COPY ./standup /usr/local/bin/standup
+COPY ./standup-linux /usr/local/bin/standup
```

Comment:
it wouldn't work without this fix

Reply:
Why did the binary name change?

Reply:
This fails on CircleCI now
```
COPY failed: stat /var/lib/docker/tmp/docker-builder262920037/standup-linux: no such file or directory
```

Reply:
I believe there is a miss-match between what the Makefile does, and what CircleCI does. `standup-linux` is the name of the binary generated by the Makefile. But it's not the one used by CircleCI. I'll revert the binary name change.

Reply:
PTAL